### PR TITLE
netmgr: improve log

### PR DESCRIPTION
**Pull request by jinliangli**
[Detail]
improve log with ulog system

[Verified Cases]
Build Pass: <py_engine_demo>
Test Pass:  <py_engine_demo>

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : a90fdb315f46162dd929d48cd46a190e6f0cf737

**Comment by skylarCai** [comment]
auto build success,number:232

**Comment by skylarCai** [comment]
autotest fail,number:1734

**Review by yanxiaoyong-ch — COMMENTED** [review]
this looks ok for me

File: components/netmgr/src/netmgr_wifi.c [hunk 1]
```diff
@@ -3457,7 +3457,7 @@ static void handle_netmgr_wifi_help_cmd()
     NETMGR_WIFI_LOGI("netmgr -t wifi -s\n");
     NETMGR_WIFI_LOGI("netmgr -t wifi -p\n");
     NETMGR_WIFI_LOGI("netmgr -t wifi -r\n");
-    NETMGR_WIFI_LOGI("netmgr -t wifi -w network={\\nssid=\"xxxxxx\"\\npassword=\"xxxxxxxx\"\\nchannel=\"1\"\\n}\\n");
+    NETMGR_WIFI_LOGI("netmgr -t wifi -w network={\\nssid=\"apple\"\\npassword=\"aos123456\"\\nchannel=\"1\"\\n}\\n");
```
Comment (skylarCai) [inline components/netmgr/src/netmgr_wifi.c:3460]: sensitive info

**Review by skylarCai — COMMENTED** [review]
security risk

**Pull request closed without merging**